### PR TITLE
Add drift detection resource

### PR DESCRIPTION
**Pull request by eranelbaz**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
Add drift detection resource

### Solution
Add it

Didn't add any integration test since I feel like it will be stupid test...

### CODE REVIEW

File: docs/resources/environment_drift_detection.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Schema
+
+### Required
+
+- **cron** (String) Cron expression for scheduled drift detection of the environment
+- **environment_id** (String) The environment's id
+
+### Optional
+
+- **id** (String) The ID of this resource.
```
Comment (GiliFaroEnv0) [inline docs/resources/environment_drift_detection.md:46]: why?

**Review by GiliFaroEnv0 — APPROVED** [review]
Looks good!
left 1 comment on the generated docs If you know why it add the id as optional?
If it really allows it you need o changed your logic using the id

File: docs/resources/environment_drift_detection.md (hunk 1 above)
Comment (GiliFaroEnv0) [reply docs/resources/environment_drift_detection.md:46]: If it really allows it you need o changed your logic using the id

Comment (eranelbaz) [reply docs/resources/environment_drift_detection.md:46]: each tf resource have `id` which is the resource `id`
in this case the actual resource `id` doesn't matter but only the environment `id` which we set

Comment (eranelbaz) [reply docs/resources/environment_drift_detection.md:46]: and it's optional since i didn't specify it as part of the schema

**Pull request merged**